tests/libvmi/factory.go

```diff
@@ -71,3 +73,16 @@ func NewAlpine(opts ...Option) *kvirtv1.VirtualMachineInstance {
 	alpineOpts = append(alpineOpts, opts...)
 	return New(RandName(DefaultVmiName), alpineOpts...)
 }
+
+// alpineMemory returns the required memory for a Cirros VM.
+// The cirros image needs 256M to boot on ARM64.
+// Ref: https://github.com/kubevirt/kubevirt/issues/6363
```

Comment:
can we remove please this comment ? 
just keep it in the PR / commit desc ?

Reply:
It was there is the original location and it seems reasonable to explain this awful need for it to exist.

I do not understand why you see it as different from any other function description.

Reply:
I think what @oshoval is trying to say is that it reads well and the comment doesn't bring much value. Note the comment is lying.

Reply:
I think the ref to the issue is the valuable part that I thought it is worth preserving.
But if two reviewers think it is not valuable, I am not going to insist. I dislike comments as well.

Reply:
Done